Ports/README.md

```diff
@@ -131,13 +131,13 @@ script simply defines some well-known variables and looks like this:
 ```bash
 #!/usr/bin/env -S bash ../.port_include.sh
 
-port="foo"
-version="1.2.3"
-useconfigure="true"
+port='foo'
+version='1.2.3'
+useconfigure='true'
 files=(
     "https://example.com/foo-${version}.tar.gz 9acd50f9a2af37e471f761c3fe7b8dea5617e51dac802fe6c177b74abf0abb5a"
 )
-depends=("bar" "baz")
+depends=('bar' 'baz')
```

Comment:
While you're here, can you put these on their own lines?

Reply:
Done!